src/components/learner-credit-management/members-tab/GroupMembersCsvDownloadTableAction.jsx

```diff
@@ -50,9 +54,13 @@ const GroupMembersCsvDownloadTableAction = ({
       }
     });
 
+    const selectedEmails = [];
+    tableInstance.selectedFlatRows.forEach((row) => selectedEmails.push(row.id));
+
     EnterpriseAccessApiService.fetchSubsidyHydratedGroupMembersData(
       subsidyAccessPolicyId,
       options,
+      selectedEmails,
```

Comment:
[nit/suggestion] we can remove the extra param here since the learner emails are added to options